.travis.yml

```diff
@@ -19,13 +19,16 @@ env:
     - SITE_DIR="$HOME/build/testing_site"
   matrix:
     - TEST=PHP_CodeSniffer
-    - TEST=PHPUnit
+    - TEST=8.5.x
+    - TEST=8.6.x
 
 matrix:
   fast_finish: true
   exclude:
     - php: 7.2
       env: TEST=PHP_CodeSniffer
+  allow_failures:
```

Comment:
Shouldn't we ad a TODO referring the issue where we remove this?

Reply:
Well we are not sure yet, there might be other dragons lurking below the surface. But I will create a dedicated issue to track this.

Reply:
We have the 8.6.x issue. That would remove this